CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## unreleased
 * BraintreePayPal
   * Add `BTPayPalRequest.userPhoneNumber` optional property
+* BraintreeVenmo
+  * Update`BTVenmoRequest` to make all properties internal and included in the initializer 
```

Comment:
I think there is a `v7` section of the CHANGELOG that was started in an earlier PR. Not sure if this PR is using the latest from `v7`.

Reply:
Oh, it looks like this was changed at some point and is further down: https://github.com/braintree/braintree_ios/blob/v7/CHANGELOG.md#700-beta1. We should move this up and rename to `unreleased (v7)` since we haven't released a beta yet and that will get pulled in automatically.